file/provider/create_volume.go

```diff
@@ -74,6 +74,7 @@ func (vpcs *VPCSession) CreateVolume(volumeRequest provider.Volume) (volumeRespo
 		}
 
 		setENIParameters(&shareTargetTemplate, volumeRequest)
+		setEITParameter(&shareTargetTemplate, volumeRequest)
```

Comment:
check here and set the mode here only, no need to create a method just for 1 line code

Reply:
Fixed